docs/guides/python/sql_on_arrow.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: docu
+title: SQL on Apache Arrow
+selected: SQL on Apache Arrow
+---
+
+# How to execute SQL on Apache Arrow
+DuckDB can query multiple different types of Apache Arrow objects. 
+
```

Comment:
For completion maybe also add an example with recordbatchreaders?

Reply:
I also added an example for this! I'm learning a fair bit about Arrow... they have a few different options for objects, huh? I'm sure you feel that more than I do...